### DISCUSSION TITLE
Doing scheduled parameter changes on inverter

### DISCUSSION THREAD

**Discussion by slizovskiy** [StephanJoubert/home_assistant_solarman#260] [General]
Hi, I wonder if this code can be deployed on some server, which would allow me to make regular scheduled changes to Deye inverter parameters.    I need to change from Zero CT export to Selling First  work Mode  for 3 hours a day (when peak price).  Deye does not allow me to do that in parameters, so, I have to change the mode manually via DeyeCloud app.  I want to automate this task.
Any ideas?
Thank you,
Sergey

**Comment by tpahula** [comment]
I am looking for solar sell option change. I am trying to avoid negative electricity prices. I can change solar sell option in Solarman Business app now. Is this change possible with solarman integration ?

**Comment by getget1980** [comment]
Hello, I need the same, to switch my inverter from "Self use mode" to "Timer mode", is it possible to write to the inverter ?
